### ISSUE TITLE
Citizen Fund - Create funds & invest in european companies

### ISSUE THREAD

**Issue by Figu3** [metacartel/dragon-quest#15]
Hey everyone, we're looking for a few profiles to help us speed up a project we've been working on lately. It's called Citizen Fund, and if you think it is as cool as we do hit us up
Telegram : FiguEU

WHAT WE DO 

Citizen Fund is a protocol that enables anyone to create a fund in order to invest in Europe. The objective is to allow non-professional investors to have the same financial and legal tools as professionals have. 
We believe it could be a great platform to counter-act the current weight of private equity on the market

WHO WE ARE

We are a bunch of lawyers who think Moloch is a game changer to reverse economic flows from corporation to people.

- I come from the world of finance, I designed the funds, the governance rules and know some people really interested in such a project.

- The second creator is a lawyer specialized in corporate law and tokenization.

WHAT WE NEED

Basically anyone who thinks they can contribute is more than welcome !

More specifically, the skills we lack are in : 

- Engineering (Smart Contracts in order to optimize Moloch v2 and create new governance rules)
- Design (basically optimizing the interface so it seems as fluid as possible for the non-crypto folks).

@FiguEU if you are interested in the project =)

Cheers

**Comment by pet3r-pan** [comment]
Sounds super cool :)
